data-r2dbc/src/test/groovy/io/micronaut/data/r2dbc/oraclexe/OracleXEManualSchemaSpec.groovy

```diff
@@ -19,19 +29,67 @@ class OracleXEManualSchemaSpec extends AbstractManualSchemaSpec implements Oracl
         return context.getBean(OracleXEPatientRepository)
     }
 
+    @Memoized
+    OracleXEJsonEntityRepository getJsonEntityRepository() {
+        return context.getBean(OracleXEJsonEntityRepository)
+    }
+
     @Override
     List<String> createStatements() {
         return Arrays.asList("CREATE SEQUENCE \"PATIENT_SEQ\" MINVALUE 1 START WITH 1 CACHE 100 NOCYCLE",
-                "CREATE TABLE \"PATIENT\" (\"NAME\" VARCHAR(255), \"ID\" NUMBER(19) NOT NULL PRIMARY KEY, \"HISTORY\" VARCHAR(1000), \"DOCTOR_NOTES\" VARCHAR(255))")
+                "CREATE TABLE \"PATIENT\" (\"NAME\" VARCHAR(255), \"ID\" NUMBER(19) NOT NULL PRIMARY KEY, \"HISTORY\" VARCHAR(1000), \"DOCTOR_NOTES\" VARCHAR(255))",
+                "CREATE TABLE \"JSON_ENTITY\" (\"ID\" NUMBER(19) NOT NULL PRIMARY KEY, \"SAMPLE_DATA\" BLOB)")
```

Comment:
And here testing BLOB which will also be read/written using Oracle Binary Mapper.